health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/BoundaryDetailsSummary.js

```
@@ -116,7 +116,7 @@ const BoundaryDetailsSummary = (props) => {
         <PopUp
           // className={"boundaries-pop-module"}
           type={"default"}
-          // heading={t("ES_CAMPAIGN_UPDATE_TYPE_MODAL_HEADER")}
+          heading= {t(( props?.hierarchyType + "_" + props?.boundaries?.type).toUpperCase())} 
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)